media/css/mozorg/home/statement.scss

```diff
@@ -0,0 +1,203 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+#promo-close {
+    background: rgba(0, 0, 0, .2) url('/media/img/notification/close.svg') center center no-repeat;
+    background-size: 20px 20px;
+    border-radius: 50%;
+    border: none;
+    display: none;
+    height: 35px;
+    padding: 6px;
+    position: absolute;
+    right: 15px;
+    top: 15px;
+    transition: background-color 0.2s ease;
+    width: 35px;
+
+    &:focus,
+    &:hover {
+        background-color: rgba(0, 0, 0, .3);
+        cursor: pointer;
+    }
+
+    // hide the 'Close' text
+    span {
+        @include visually-hidden;
+    }
+
+    // only display when JS is available
+    .js & {
+        display: block;
+    }
+}
+
+#promo-statement {
+    background: #fff;
+    color: #000;
+    padding: 40px 20px;
+    min-height: 100vh;
+
+    // hide by default if JS is available to avoid flicker
+    // (if visitor previously dismissed)
+    .js & {
+        display: none;
+    }
+
+    h1,
+    p {
+        @include font-size-level4;
+        font-family: Monaco, monospace;
+        font-weight: normal;
+        margin-bottom: 1.25em;
+    }
+
+    a {
+        color: #00f;
+    }
+
+    &:before {
+        @include background-size(100px, 32px);
+        background: url('/media/img/pebbles/moz-wordmark-dark-reverse.svg') no-repeat;
+        content: '';
+        display: inline-block;
+        height: 32px;
+        left: 20px;
+        position: absolute;
+        top: 20px;
+        width: 100px;
+    }
+
+    .content-wrapper {
+        margin: 0 auto;
+        margin-top: 20vh;
+        max-width: 53ch;
+        padding-top: 10px;
+    }
+}
+
+/* with graditiude to http://lea.verou.me/2012/02/simpler-css-typing-animation-with-the-ch-unit/*/
+$width: 50;
+$text: #000;
+$bg: #fff;
+$cursor-speed: 20; // characters per second
+$duration: $width / $cursor-speed * 1s;
+$cursor-duration: $duration / $width;
+$cursor-wait-duration: 1s;
+$cursor-wait-iteration: 2;
+$paragraph-lengths: (24, 23, 86, 34);
+
+@keyframes letters {
+    // animation, go from left to right
+    from {
+        left: 0;
+    }
+    to {
+        left: $width + ch;
+    }
+}
+
+@keyframes lines {
+    // go from top to bottom
+    from {
+        top: 0;
+    }
+    to {
+        top: 100%;
+    }
+}
+
+@keyframes cursor-type {
+    50% {
+        border-color: #000;
+    }
+}
+
+@keyframes cursor-wait {
+    0% {
+        box-shadow: inset 1px 0 0 0 rgba(0, 0, 0, 0);
+    }
+    1% {
+        box-shadow: inset 1px 0 0 0 rgba(0, 0, 0, 1);
+    }
+    40% {
+        box-shadow: inset 1px 0 0 0 rgba(0, 0, 0, 1);
+    }
+}
+
+.type {
+    position: relative;
+    font-family: Consolas, Monaco, monospace; // must be monospace
+    width: $width + ch;
```

Comment:
50ch = about 540px at this font size and it's overflowing on small screens. Probably needs some media query trickery to adjust the width at different breakpoints, though I'm not sure how that might complicate the typing animation.

Reply:
Yeah, I havn't addressed mobile at all yet :( I might just not give them any animation.

Reply:
That was my initial thought, too. Put the whole animation treatment behind a media query and just show the regular text on mobile. Probably the simplest rather than trying to adjust the size and keep the typing.

Reply:
It actually wouldn't be that hard, I just have to make my loop into a mixin... but time :/